src/util/AbstractMemory/CalculationUtil.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (July of 2023,
+// schlegea@informatik.uni-freiburg.de)
+
+#pragma once
+
+#include <cmath>
+#include <cstddef>
+
+#include "util/ConstexprUtils.h"
+
+// Just the number of bytes per memory unit.
+constexpr size_t numberOfBytesPerKB = ad_utility::pow<size_t>(2, 10);
+constexpr size_t numberOfBytesPerMB = ad_utility::pow<size_t>(2, 20);
+constexpr size_t numberOfBytesPerGB = ad_utility::pow<size_t>(2, 30);
+constexpr size_t numberOfBytesPerTB = ad_utility::pow<size_t>(2, 40);
+constexpr size_t numberOfBytesPerPB = ad_utility::pow<size_t>(2, 50);
```

Comment:
We'll have to discuss if we want binary or decimal MB, so either MB are 1000 and MiB are 1024, or we just don't care.
I'll talk to @hannahbast about this, also about whether we want the suffixes to be `kB` or `KB`:)

Reply:
Done, see [346b0a7](https://github.com/ad-freiburg/qlever/pull/1032/commits/346b0a711751658aa227883b31a3683a6a4bde51).